packages/dropdowns/src/elements/Menu/Items/HeaderIcon.spec.tsx

```diff
@@ -0,0 +1,83 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import { render } from 'garden-test-utils';
+import { Dropdown, Trigger, Menu, HeaderItem, HeaderIcon } from '../../..';
+import { DEFAULT_THEME } from '@zendeskgarden/react-theming';
+
+const ExampleSvg = React.forwardRef<SVGSVGElement, {}>((props, ref) => (
+  <svg ref={ref} {...props} />
+));
```

Comment:
Not a blocker, but I was curious if there was a particular reason this component forwards a `ref`? The following appears to work and would yield the same test results:

```jsx
const ExampleSvg = () => <svg />;
```

Reply:
I think I just got in the habit of using `forwardRef` 😄 Removed.